plugin/geoip/README.md

```diff
@@ -16,35 +18,54 @@ import (
 // ...
 if getLongitude := metadata.ValueFunc(ctx, "geoip/longitude"); getLongitude != nil {
     if longitude, err := strconv.ParseFloat(getLongitude(), 64); err == nil {
-		// Do something useful with longitude.
-	}
+        // Do something useful with longitude.
+    }
 } else {
     // The metadata label geoip/longitude for some reason, was not set.
 }
 // ...
 ```
 
 ## Databases
+
 The supported databases use city schema such as `City` and `Enterprise`. Other databases types with different schemas are not supported yet.
 
 You can download a [free and public City database](https://dev.maxmind.com/geoip/geolite2-free-geolocation-data).
 
 ## Syntax
-```txt
+
+```text
 geoip [DBFILE]
 ```
+
+or
+
+```text
+geoip [DBFILE] {
+    edns-subnet-processing
```

Comment:
```suggestion
    [edns-subnet-processing]
```

Reply:
Another _very_ minor thing: `edns-subnet-processing` is kind of a long option name.  e.g. I think just `edns-subnet` would be sufficient.

Reply:
`edns-subnet-processing` is taken from PowerDNS options, verbatim. We're not tied to pdns though :)